tests/tasks/test_bfdtask.py

```diff
@@ -10,34 +9,6 @@
 from zino.tasks.bfdtask import BFDTask
 
 
-class TestConvertAddress:
-    def test_converts_bytes_to_correct_ipv6_address(self):
-        parsed_address = BFDTask._convert_address(
-            b"\x00\x00\x00\x00\x00\x00\x00\x00\00\x00\x00\x00\x00\x00\x00\x01",
```

Comment:
This case could be added to the utils test

Reply:
the new parse_ip func wouldnt handle bytes like that. the old _convert_address function took bytes since thats what Iconverted "0x7f000001" format into first, then from bytes to IPAddress. I dont think anything else IP related use bytes like this

Reply:
Ah, goody then, we can always add to the function if we happen to need that functionality

Reply:
the standard ip_address should handle bytes just fine anyways :)